gapic/src/main/com/google/gapid/perfetto/models/SliceTrack.java

```diff
@@ -242,6 +308,43 @@ public ThreadInfo getThread() {
         return thread;
       }
     }
+
+    public static class GpuSlice extends Slice {
+      private final RenderStageDebugMarkers debugMarkers;
```

Comment:
This should not be named RenderStageDebugMarkers.  The vulkan handles are important whether we have the debug markers are available or not.
Something like GpuInfo would be a better name, and it compliments ThreadInfo.

Reply:
GpuInfo is already used as a class for obtaining the queues. Check GpuInfo.java under perfetto/models. Maybe something like a RenderStageInfo would do.

Reply:
Done.